include/linker/sections.h

```diff
@@ -31,8 +31,8 @@
 #define _UNDEFINED_SECTION_NAME undefined
 
 /* Interrupts */
-#define _IRQ_VECTOR_TABLE_SECTION_NAME	.gnu.linkonce.irq_vector_table
-#define _SW_ISR_TABLE_SECTION_NAME	.gnu.linkonce.sw_isr_table
+#define _IRQ_VECTOR_TABLE_SECTION_NAME	.gnu.linkonce.irq_vector_table*
+#define _SW_ISR_TABLE_SECTION_NAME	.gnu.linkonce.sw_isr_table*
```

Comment:
There was a question from @sebo:
> Do we know why mwdt is generating the . suffix and gcc is not?

could you answer that ?

Reply:
@tejlmand , @SebastianBoe 

> There was a question from @sebo:
> 
> > Do we know why mwdt is generating the . suffix and gcc is not?
> 
> could you answer that ?

@tejlmand, @SebastianBoe as @evgeniy-paltsev mentioned in the commit message here https://github.com/zephyrproject-rtos/zephyr/pull/22668/commits/8169f2d586ba973145a7bbb8789eda44628e9e86 that's a behavior of MetaWare toolchain and it differs from GCC. I'm not a compiler expert and not sure if there's an established standard for section names generation. If there's such a standard I'll happily point to it our MetaWare developers with request to accommodate it.

Reply:
thanks